test/Microsoft.IdentityModel.Tests/KeyingMaterial.cs

```diff
@@ -225,6 +225,11 @@ public static RsaSecurityKey RsaSecurityKey2
         public static byte[] DefaultSymmetricKeyBytes_128 = Convert.FromBase64String(DefaultSymmetricKeyEncoded_128);
         public static SymmetricSecurityKey DefaultSymmetricSecurityKey_128 = new SymmetricSecurityKey(DefaultSymmetricKeyBytes_128) { KeyId = "DefaultSymmetricSecurityKey_128" };
 
+        public static string DefaultSymmetricKeyEncoded_192 = "hf+ctyfSxzcVMn5B5D+u4It4/HHjnh/E";
+        public static byte[] DefaultSymmetricKeyBytes_192 = Convert.FromBase64String(DefaultSymmetricKeyEncoded_192);
+        public static SymmetricSecurityKey DefaultSymmetricSecurityKey_192 = new SymmetricSecurityKey(DefaultSymmetricKeyBytes_192) { KeyId = "DefaultSymmetricSecurityKey_192" };
+        
+
```

Comment:
nit: one empty too many

Reply:
addressed with: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1025/commits/86c08099b5a6a16659279f788edf99625bbb7151